pages/new-collective-page.js

```diff
@@ -125,7 +124,7 @@ class NewCollectivePage extends React.Component {
     const collective = data && data.Collective;
     return (
       <Page {...this.getPageMetaData(collective)} withoutGlobalStyles>
-        <GlobalStyles />
+        <GlobalStyles smooth={get(data.Collective, 'path') && true} />
```

Comment:
It's not a bad workaround, maybe we can write on the `id` instead as it's the only field we'll always keep for sure

```suggestion
        <GlobalStyles smooth={Boolean(get(collective, 'id'))} />
```